### PR TITLE
Added pulsar-rest-api blogpost

### PR DESCRIPTION
**Pull request by kris-lab**

### CODE REVIEW

**Comment by kris-lab** [comment]
@njam please review

**Comment by njam** [comment]
@vogdb please review

File: _posts/2015-06-23-pulsar-rest-api.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+tags: [pulsar, rest, api, hipchat, capistrano, deployment, devops, automation, nodejs]
+---
+
+If you work in scale, If you manage multiple applications in multiple environments, If you automate any tasks using Capistrano then try a handy solutions which will improve and extend your workbench.
+
+<!--more-->
+
+### Real-life scenario
+
+There are many automated tasks in everyday work. It is crucial to distribute the management of that tasks between not only DevOps anymore.
```
Comment (vogdb) [inline _posts/2015-06-23-pulsar-rest-api.markdown]: * of ~~that~~ those tasks?

File: _posts/2015-06-23-pulsar-rest-api.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+If you work in scale, If you manage multiple applications in multiple environments, If you automate any tasks using Capistrano then try a handy solutions which will improve and extend your workbench.
+
+<!--more-->
+
+### Real-life scenario
+
+There are many automated tasks in everyday work. It is crucial to distribute the management of that tasks between not only DevOps anymore. 
+The idea was for distributing small tasks on the Dev teams as well. One of the most important task is the deployment process
```
Comment (vogdb) [inline _posts/2015-06-23-pulsar-rest-api.markdown]: to distribute small tasks on Dev teams as well?

File: _posts/2015-06-23-pulsar-rest-api.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+If you work in scale, If you manage multiple applications in multiple environments, If you automate any tasks using Capistrano then try a handy solutions which will improve and extend your workbench.
+
+<!--more-->
+
+### Real-life scenario
+
+There are many automated tasks in everyday work. It is crucial to distribute the management of that tasks between not only DevOps anymore. 
+The idea was for distributing small tasks on the Dev teams as well. One of the most important task is the deployment process 
+of new release of every applications in multiple environments. It requires a very high level of permissions
```
Comment (vogdb) [inline _posts/2015-06-23-pulsar-rest-api.markdown]: of new app release in multiple environments?

File: _posts/2015-06-23-pulsar-rest-api.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+### Real-life scenario
+
+There are many automated tasks in everyday work. It is crucial to distribute the management of that tasks between not only DevOps anymore. 
+The idea was for distributing small tasks on the Dev teams as well. One of the most important task is the deployment process 
+of new release of every applications in multiple environments. It requires a very high level of permissions 
+and orchestration of many actions by experienced DevOps. As we use [Pulsar](https://github.com/nebulab/pulsar) which 
+is built on top of [Capistrano](https://github.com/capistrano/capistrano) we decided to extend that tool with REST interface. 
+It allows us to talk easily over HTTP to our tasks. We do it via [Hubot](https://hubot.github.com/). 
+This simple piece of software lets us to integrate [HipChat](https://www.hipchat.com/) (our communication channel) 
+with simple BOT. It listens for our commands and trigger actions over pulsar-rest-api into pulsar itself.
```
Comment (vogdb) [inline _posts/2015-06-23-pulsar-rest-api.markdown]: over [pulsar-rest-api](https://github.com/cargomedia/pulsar-rest-api) ?

Comment (vogdb) [inline _posts/2015-06-23-pulsar-rest-api.markdown]: Ahh, sorry. I see you've added the link below. It's ok then to not have the link here.

**Comment by vogdb** [comment]
Is it ok to show IP addresses and the sequence of steps in the dashboard screenshot?

Besides that and the line notes, looks good.

**Comment by vogdb** [comment]
@kris-lab ?

**Comment by kris-lab** [comment]
@vogdb please re-review
- IPs are private so it is safe (there are not visible fully anyway)

**Comment by vogdb** [comment]
lgtm then!

**Pull request merged**